### PR TITLE
Widgetkotlin

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

File: android/app/src/androidTest/java/io/bluewallet/bluewallet/DetoxTest.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 @RunWith(AndroidJUnit4.class)
 @LargeTest
-public class DetoxTest {
+public class    DetoxTest {
```
Comment (Overtorment) [inline android/app/src/androidTest/java/io/bluewallet/bluewallet/DetoxTest.java:16]: unnecessary change

File: ios/BlueWalletTests/PriceViewTests.swift [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+//
+//  Created by Marcos Rodriguez on 7/10/24.
+//  Copyright © 2024 BlueWallet. All rights reserved.
+//
+
+import XCTest
+import SwiftUI
+import WidgetKit
+
+@testable import BlueWallet
```
Comment (Overtorment) [inline ios/BlueWalletTests/PriceViewTests.swift:13]:
```
GitHub Actions
/ build

no such module 'BlueWallet'
```

**Review by Overtorment — CHANGES_REQUESTED** [review]
did ios build fail..?

**Review by Overtorment — CHANGES_REQUESTED** [review]
image

**Review by Overtorment — APPROVED** [review]

**Pull request merged**